backend/src/types.ts

```diff
@@ -304,7 +304,7 @@ export type UserChoice = ICourseRange | IOrCourse;
 export interface Major {
   name: string;
   sections: string[];
-  sectionMap: { [key: string]: ANDSection | ORSection };
+  sectionMap: { [key: string]: ANDSection | ORSection | RANGESection };
```

Comment:
At the meeting, you mentioned renaming this to something, will that be updated here?